rotkehlchen/types.py

```diff
@@ -551,7 +551,7 @@ class Location(DBCharEnumMixIn):
     KUCOIN = 23
     BALANCER = 24
     LOOPRING = 25
-    FTX = 26
+    FTX = 26  # FTX is dead but we keep the location to allow users to import FTX events via csv
```

Comment:
It's not just for import.

1. Trades that happened, happened.
2. Locations can't be deleted. It's an enum that's also saved in the DB. This value is now used

```suggestion
    FTX = 26  # FTX is dead but we keep the location for historical reasons
```